Fixed install instructions

SengerM opened the pull request.

aWZHY0yQH81uOYvH commented:
Is this not just the same instructions but with a slightly different example path? (and no extraneous trailing quote)

SengerM commented:
> Is this not just the same instructions but with a slightly different example path? (and no extraneous trailing quote)

Yes, but
1. Removed the "simple install" method that does not work (I tried in Fedora 40 and in Ubuntu 24.04, both failed for me).
2. Changed path to the one inside the home directory so there is no need for `sudo` and is also in line with the instructions [here](https://docs.openems.de/install/clone-build-install.html)
3. For me it only worked after adding the blank space after `-I`, `-L` and `-R`.

In any case, [#239 in the openEMS-project repo](https://github.com/thliebig/openEMS-Project/pull/239) and the linked PRs ([linked 1](https://github.com/thliebig/openEMS/pull/151), [linked 2](https://github.com/thliebig/CSXCAD/pull/45)) is a much better option than this PR.
